### PR TITLE
BaSP-MR-75: Create list and delete members

### PR DESCRIPTION
**Pull request by FrancoLelli**
I create a list of members and the functionality of delete

### CODE REVIEW

**Comment by FrancoLelli** [comment]
[MEMBER - List/delete table](https://trello.com/c/ZeOxGHZw/75-member-list-delete-table)

File: src/Components/Members/DeleteMember/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import styles from './delete-member.module.css';
+import ModalConfirm from '../../Modals/ModalConfirm';
+import ModalSuccess from '../../Modals/ModalSuccess';
+
+const DeleteMember = ({ memberId, onDeleteMember }) => {
+  const [modalDeleteConfirmOpen, setModalDeleteConfirmOpen] = useState(false);
+  const [modalSuccessConfirmOpen, setModalSuccessConfirmOpen] = useState(false);
+  const [idToDelete, setIdToDelete] = useState('');
```
Comment (NicoCagnina) [inline src/Components/Members/DeleteMember/index.js:9]: I think we should initialize it as null

File: src/Components/Members/index.js [hunk 2]
```diff
@@ -1,25 +1,53 @@
...
+    getMembers();
   }, []);
 
+  const memberDelete = async (memberId) => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/member/${memberId}`, {
+        method: 'DELETE'
+      });
+      console.log(response);
+      setMembers([...members.filter((member) => member._id !== memberId)]);
```
Comment (NicoCagnina) [inline src/Components/Members/index.js:28]: nice!

File: src/Components/Members/index.js [hunk 3]
```diff
@@ -1,25 +1,53 @@
...
+        <h2 className={styles.letterColour}>Members</h2>
+        <div className={styles.addContainer}>
+          <img
+            className={styles.imgSize}
+            src={`${process.env.PUBLIC_URL}/assets/images/btn-add.png`}
+          />
+          <p>Add Member</p>
+        </div>
       </div>
+      {members.length === 0 ? (
```
Comment (NicoCagnina) [inline src/Components/Members/index.js:46]: can't we just validate `members.length`? Since 0 is falsy it should work in the same way

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

**Review by MaateoC — APPROVED** [review]

**Review by CristianLotorto — APPROVED** [review]
it seems to be all ok

**Review by NicoCagnina — APPROVED** [review]

**Pull request merged**